src/modules/feature-modules/accessor/pages/actions/actions-advanced-search.component.html

```diff
@@ -104,7 +104,7 @@ <h2 class="nhsuk-heading-xs nhsuk-u-margin-0">Search by innovation name</h2>
                       {{ item.name }}
                     </a>
                   </div>
-                  <theme-notification-tag label="dot" *ngIf="item.notifications && item.notifications.count > 0"></theme-notification-tag>
+                  <theme-notification-tag label="dot" *ngIf="item.notifications && item.notifications > 0"></theme-notification-tag>
```

Comment:
this change seems odd, second condition is always true if first one

Reply:
changed... thanks..